k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -2204,12 +2188,17 @@ public void handleMessage(android.os.Message msg) {
                     break;
                 case MSG_SAVED_DRAFT:
                     draftId = (Long) msg.obj;
-                    FeedbackTools.showLongFeedback(getRootView(),
-                            getString(R.string.message_saved_toast));
+                    finishWithSnackBar(() -> FeedbackTools.showLongFeedback(getRootView(),
+                            getString(
+                                    isInvite
```

Comment:
is this flag only used in this case? I don't understand well the message, invite sent and draft saved, why draft? Is that the email the user was about to write that is saved as draft?

Reply:
yes, just there is also the case of saving as draft when sending an invite. Then we want to know that we sent an invite in order to show a custom feedback to the user, like "invite sent and message saved as draft", instead of just "message saved as draft".

Reply:
ok I see